README.md

```diff
@@ -125,27 +124,36 @@ The installation was tested on Windows Subsystem for Linux **V1** ([WSL](https:/
     mamba install -c robotology compilers cmake
     ```
 
-### Install the Repository
+### Install the Repository: One line Installer
 
-6. Clone the `yarp-openmct` repository into `<yarp-openmct-root-folder>` folder
+For most users, the one line installation procedure is recommended, directly installing the remote repository URL via the **npm** package manager:
+
+6. In a location of your choice, run the **npm install** command directly from the repository remote URL 
     ```
-    git clone https://github.com/dic-iit/yarp-openmct.git <yarp-openmct-root-folder>
+    npm install https://github.com/dic-iit/yarp-openmct.git --global-style true
     ```
+This command clones the repository with the protocol `git+http`, installs all the source files (without the `.git`, `.gitignore` nor other version control files) under the folder `./node_modules/yarp-openmct` and installs the dependencies following the manifest `./node_modules/yarp-openmct/package.json`.
+
+Always make sure you are selecting the proper **node.js** version (v14.17.0) before running an **npm** command.
+
+**Note**: The option `--global-style` causes **npm** to install the `yarp-openmct` package into your local `./node_modules` folder with the same layout it uses with the global `node_modules` folder (find it with `npm root -g`). Only `yarp-openmct`'s direct dependencies will show in `./node_modules`, while everything they depend on will be flattened in their own `node_modules` folders. This obviously will eliminate some deduping, but in counterpart it allows for instance the `cmake-js` compilation step of the dependency package `yarp.js` to consistently find all of its own dependencies.
```

Comment:
Not blocking the PR, but I am not sure the long explanation in this place (the section is not for developers, but for naive users) is helping. I am afraid most users will be confused by this technical explanation.

Reply:
Good point, I will remove it in my next PR.

Reply:
Since this is introduced in this PR I think it is better to remove it before merging.

Reply:
Sorry @RiccardoGrieco , I had missed your last comment, and you're right, for the sake of context. In this case, I had already included it in the next changes to be PRed.